.gitmodules

```diff
@@ -38,3 +38,7 @@
   path = CDEPS-interface/CDEPS
   url = https://github.com/NOAA-EMC/CDEPS
   branch = develop
+[submodule "GOCART"]
+  path = GOCART
+  url = https://github.com/GEOS-ESM/GOCART
+  branch = feature/rmontuoro/ufs-integration
```

Comment:
Is this branch going to be merged to the main branch? I do not see a PR. Or is UFS going to point to what's effectively a private (personal) branch?

Reply:
This branch (`feature/rmontuoro/ufs-integration`) does not exists in `GEOS-ESM/GOCART` repo. At least I do not see it.

Reply:
Not sure why the feature branch is still included in `.gitmodules`. The submodule points to the main GOCART branch.

Reply:
Fixed in 7dac1bdd888351aa809273b1663a83851ba1803a